src/queue.c

```diff
@@ -0,0 +1,101 @@
+#include "queue.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#define TRUE 1
+#define FALSE 0
+
+struct node
```

Comment:
You seem to be declaring node and queue twice. Once in the header and once in the source. Just declare them in the header. If you declare them in the source they won't (easily) be visible to other files in our repo.